server/bin/pbench-unpack-tarballs.sh

```diff
@@ -4,38 +4,17 @@
 # This script is the first part of the pipeline that processes pbench
 # results tarballs.
 
-# First stage:  pbench-unpack-tarballs looks in all the TODO
-#               directories, unpacks tarballs, checks MD5 sums and
-#               moves the symlink from the TODO subdir to the
-#               TO-COPY-SOS subdir.  It runs under cron once a minute
-#               in order to minimize the delay between uploading the
-#               results and making them available for viewing over the
-#               web.
-
-# Second stage: pbench-copy-sosreports looks in all the TO-COPY-SOS
-#               subdirs, extracs the sos report from the tarball and
-#               copies it to the VoS incoming area for further
-#               processing. Assuming that all is well, it moves the
-#               symlink from the TO-COPY-SOS subdir to the TO-INDEX
-#               subdir.
-
-# Third stage:  pbench-index looks in all the TO-INDEX subdirs and
-#               calls the pbench-results-indexer script to index the
-#               results tarball into ES. It then moves the symlink from
-#               the TO-INDEX subdir to the DONE subdir.
-
-# assumptions:
-# - this script runs as a cron job
-# - tarballs and md5 sums are uploaded by move/copy-results to
-#   $ARCHIVE/$(hostname -s) area.
-# - move/copy-results also makes a symlink to each tarball it uploads
-#   in $ARCHIVE/TODO.
-
-# This script loops over the contents of $archive/TODO, verifies the md5
-# sum of each tarball, and if correct, it unpacks the tarball into
-# .../incoming/$(hostname -s)/.  If everything works, it then moves the
-# symlink from $ARCHIVE/TODO to $ARCHIVE/TO-COPY-SOS.
-
+# `pbench-unpack-tarballs` looks in all the TO-UNPACK or TO-RE-UNPACK
+# directories, unpacks tarballs, and moves the symlink from the TO-[RE-]UNPACK
+# subdir to the UNPACKED subdir.  It runs under cron once a minute in order to
+# minimize the delay between uploading the results and making them available for
+# viewing via the web server.
+
+# This script loops over the contents of ${ARCHIVE}/<controller>/TO-[RE-]UNPACK
+# and unpacks the tarball into .../incoming/<controller>/, establishing the
+# proper .../results and .../users symlinks.  If everything works, it then moves
+# the tar ball symlink from ${ARCHIVE}/TO-[RE-]UNPACK to ${ARCHIVE}/UNPACKED to
+# mark the process complete.
```

Comment:
Interesting that the comment was perhaps never updated when the "bucket" concept was introduced. Since you bothered to clean it up at all, it wouldn't be a bad idea to add a mention.